mina-daemon/templates/configmap.yaml

```diff
@@ -1,4 +1,3 @@
-{{- if .Values.deployment.storeBlocks.aws.enabled }}
```

Comment:
This configmap is now mandatory as it includes a new entrypoint.sh.
The above condition is pushed in a block further down